src/webserver/Dynamic.jl

```diff
@@ -377,6 +381,7 @@ responses[:reshow_cell] = function response_reshow_cell(🙋::ClientRequest)
     end
     run = WorkspaceManager.format_fetch_in_workspace((🙋.session, 🙋.notebook), cell.cell_id, ends_with_semicolon(cell.code), (parse(PlutoRunner.ObjectID, 🙋.body["objectid"], base=16), convert(Int64, 🙋.body["dim"])))
     set_output!(cell, run)
+    set_dependencies!(cell, 🙋.notebook)
```

Comment:
I am not sure if this is needed here.
Is it possible that the cell value actually changes in `response_reshow_cell`? If not, this could be removed for performance reasons.

Reply:
Not needed 🌈

Reply:
Thanks, is removed.